src/modules/Dashboard/Components/Dashboard/Dashboard.tsx

```diff
@@ -1,7 +1,157 @@
-import React from 'react'
+import { useEffect, useState } from "react";
+import { Box, Typography } from "@mui/material";
+import WorkTwoToneIcon from "@mui/icons-material/WorkTwoTone";
+import axios from "axios";
+import { getDashboard } from "../../../../services/urls";
+import CircleChart from "../../../Charts/Chart";
+import UsersChart from "../../../Charts/UsersChart";
 
-export default function Dashboard() {
-  return (
-    <div>Dashboard</div>
-  )
+interface BookingData {
+  completed: number;
+  pending: number;
+}
+
+interface UsersData {
+  admin: number;
+  user: number;
 }
+
+export default function Home() {
+  const [rooms, setRooms] = useState(0);
+  const [facilities, setFacilities] = useState(0);
+  const [ads, setAds] = useState(0);
+  const [booking, setBooking] = useState<BookingData>({
+    completed: 0,
+    pending: 0,
+  });
+  const [users, setUsers] = useState<UsersData>({ admin: 0, user: 0 });
+
+  const getDashboardData = async () => {
+    try {
+      const response = await axios.get(getDashboard, {
+        headers: { Authorization: localStorage.getItem("token") },
+      });
+      console.log(response?.data?.data);
+      setRooms(response?.data?.data.rooms);
```

Comment:
Since all of these data is coming from one api, we can make one state and from it derive the other values. This will prevent rendering the component unnecessarily.